docs/ako_operator.md

```diff
@@ -83,7 +83,7 @@ The following table also lists the configurable fields in the `AKOConfig` object
 | **Parameter** | **Description** | **Default** |
 | --- | --- | --- |
 | `replicaCount` | Specify the number of replicas for AKO StatefulSet | 1 |
-| `imageRepository` | Specify docker-registry that has the ako image | projects.registry.vmware.com/ako/ako:1.11.1 |
+| `imageRepository` | Specify docker-registry that has the ako image | projects.packages.broadcom.com/ako/ako:1.13.1 |
```

Comment:
Since this is ako-operator related document, should we update the image tag to 1.13.1 now?